plugins/destination/duckdb/client/client.go

```diff
@@ -56,12 +57,11 @@ func New(ctx context.Context, logger zerolog.Logger, spec []byte, _ plugin.NewCl
 			internalPlugin.Team, internalPlugin.Kind, internalPlugin.Name, strings.TrimPrefix(internalPlugin.Version, "v"), runtime.GOOS, runtime.GOARCH)
 	}
 
-	c.connector, err = duckdb.NewConnector(c.spec.ConnectionString, nil)
+	dbconnector, err := duckdb.NewConnector(c.spec.ConnectionString, nil)
```

Comment:
did you forget to init `c.conn` somewhere around here?

Reply:
No, c.conn is for appender.